dev/io.openliberty.org.jboss.narayana.rts/bnd.bnd

```diff
@@ -0,0 +1,37 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: io.openliberty.org.jboss.narayana.rts; singleton:=true
+                     
+Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
+
+# We will whittle this down once it is working
+Export-Package: \
+  io.narayana.lra.*; version=5.10.5, \
+  io.narayana.lra; version=5.10.5
+
+# TODO what should this be?
+#WS-TraceGroup: REACTIVESTREAMS
```

Comment:
I don't think it would help to declare a trace group for this bundle since trace groups unfortunately don't work for third party code.